tsconfig.json

```diff
@@ -1,40 +1,62 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es6",
-    "module": "commonjs",
+    "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
     "lib": [
       "es2017",
```

Comment:
Your es file is using es2020, can anything be adjusted here?

Reply:
So all I did for this file was just reverted to the tsconfig I had long ago and that worked lol.

I don't plan on making any future tweaks to this file in this PR as least.